lib/special_route_publisher.rb

```diff
@@ -9,7 +9,6 @@ def initialize(publisher_options)
   def publish(route_type, route)
     @publisher.publish(
       route.merge(
-        format: "special_route",
```

Comment:
Could you add some context on why `special_route` is no longer needed here?

Reply:
Ah yeah - format isn't really a thing any more. It was sort of a mixture of doc type and schema back in the day.  

The special route publisher defaults the `document_type` and `schema` now unless you override them --> https://github.com/alphagov/gds-api-adapters/blob/674c2faab55e0ad6e6958c747ab1a66b564a4b59/lib/gds_api/publishing_api/special_route_publisher.rb#L18-L19.  

We're OK with the doc type being answer, because that'll get rummager on our side, but the schema of our content item should be special_route because that's more appropriate.

Reply:
Makes sense - thanks for explaining!